Mixpanel alternatives

andyvan-ph opened the pull request:
## Changes

No longer a work in progress. Ready for line edits etc.

## Checklist
- [x] Titles are in [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)
- [x] Feature names are in **[sentence case too]([https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case))**
- [x] Words are spelled using American English
- [x] I have checked out our [style guide](https://github.com/PostHog/posthog.com/blob/master/STYLEGUIDE.md)
- [x] If I moved a page, I added a redirect in `vercel.json`

joethreepwood commented on contents/blog/mixpanel-alternatives.md:
```diff
@@ -0,0 +1,63 @@
+  - Comparisons
+---
+
+INTRO INTRO INTRO 
```
Needs an intro here, just fyi

joethreepwood commented on contents/blog/mixpanel-alternatives.md:
```diff
@@ -0,0 +1,63 @@
+
+According to [data compiled by Jason Packer](https://www.linkedin.com/posts/jhpacker_just-updated-the-popularity-numbers-on-my-activity-7112462135120601088-YLdh/), an independent consultant:
+
+- PostHog is deployed by 2,365 (0.2%) of the top 1 million websites in 2023, up +327% from 554 the previous year. It's the fastest growing product analytics tool on the market.
```
I'd flip this to emphasize the strength, not the math. Improving 300% from a low number isn't impressive. Being the fastest growing tool on the market, is.

joethreepwood commented on contents/blog/mixpanel-alternatives.md:
```diff
@@ -0,0 +1,63 @@
+
+### How popular is PostHog?
+
+According to [data compiled by Jason Packer](https://www.linkedin.com/posts/jhpacker_just-updated-the-popularity-numbers-on-my-activity-7112462135120601088-YLdh/), an independent consultant:
```
Worth reaching out to Packer, for quotes, content, sources, etc?

joethreepwood commented on contents/blog/mixpanel-alternatives.md:
```diff
@@ -0,0 +1,63 @@
+
+### Why do companies use PostHog?
+
+According to [reviews on G2.com](https://www.g2.com/products/mixpanel/reviews), [summarized by Senja](https://senja.io/tools/review-mining/share?s=NOmOO5mS), companies use PostHog because:
```
Personally, I don't like the addition of Senja here. Better to say that reviews show that it has these three main perks, among others?

I'd also add a section on WHO uses PostHog. Emphasize ICP, and I can back it up with G2 data if needed. 

Of course, if you're going to bring G2 into the mix then it raises the question of whether you want to use their reports, badges, etc. If not, is G2 the right platform to bring up and would we be better pointing to the case studies for social proof instead? Case studies are probably a good addition here anyway.

andyvan-ph replied:
- RE: Senja, I think it's useful to show we're not just making up our own points here. I can probably de-emphasize so it's not so prominent, though.

- Will look at ways to work customer stories in, but I'm trying to keep the top-level stuff neutrally sourced.

joethreepwood commented:
Main thoughts at the highest level: 
- I think it's fine for this to be longer, up to 750 words, and for PostHog to be the longest entry
- I think adding a table to communicate some of the features in comparison, either repeatedly or uniquely for PostHog, will help you cut length

ivanagas reviewed (commented):
My thoughts on structure:
1. Summary
2. Key features
3. Who uses product (use cases, target users, customers)
4. How it compares
5. How popular

I think why companies overlaps a lot with both how it compares and key features, could be wrapped into who uses product, but provides more options for other alternatives. I think it is important to think about how easy it will be to get this data for other options.

andyvan-ph commented:
@joethreepwood @ivanagas This is 90% there, but would like some feedback on some changes.

Specifically:
- I've changed up the format + order of sections in the PostHog item. The "How does X compare?" is now a table and earlier, just the balance out the wordiness a little. If people think this works, I'll adapt this approach with the rest.

- Added a "Bottom line" to the end to the end of each alternative. Used a boxout to add emphasis / make it more scanable.

joethreepwood commented on contents/blog/mixpanel-alternatives.md:
```diff
@@ -0,0 +1,322 @@
+
+### How does PostHog compare to Mixpanel?
+
+PostHog's built-in features make it the obvious choice if you like the idea of consolidating around one platform. Mixpanel's offers plentiful third-party integrations for features it doesn't have, but this creates a messy workflow for teams.
```
```suggestion
PostHog's built-in features make it the obvious choice if you like the idea of consolidating around one platform, and it can be extended further thanks to it being open source. Mixpanel offers plentiful third-party integrations for features it doesn't have, but this creates a messy workflow for teams. 
```

joethreepwood commented on contents/blog/mixpanel-alternatives.md:
```diff
@@ -0,0 +1,322 @@
+
+### How popular is PostHog?
+
+According to [data compiled by Jason Packer](https://www.linkedin.com/posts/jhpacker_just-updated-the-popularity-numbers-on-my-activity-7112462135120601088-YLdh/), an independent analytics consultant, _PostHog is the fastest growing Mixpanel alternative_ in the market,
```
```suggestion
According to [data compiled by Jason Packer](https://www.linkedin.com/posts/jhpacker_just-updated-the-popularity-numbers-on-my-activity-7112462135120601088-YLdh/), an independent analytics consultant, _PostHog is the fastest growing Mixpanel alternative_ in the market.
```

joethreepwood commented on contents/blog/mixpanel-alternatives.md:
```diff
@@ -0,0 +1,322 @@
+
+According to [data compiled by Jason Packer](https://www.linkedin.com/posts/jhpacker_just-updated-the-popularity-numbers-on-my-activity-7112462135120601088-YLdh/), an independent analytics consultant, _PostHog is the fastest growing Mixpanel alternative_ in the market,
+
+**PostHog** is deployed by 2,365 (0.2%) of the top 1 million websites in 2023, up +327% from 554 the previous year. **Mixpanel** is deployed by 5,218 (0.5%), down -17% from 6,295 the previous year.
```
As of...

andyvan-ph replied:
```suggestion
As of Jul 2023, **PostHog** is deployed by 2,365 (0.2%) of the top 1 million websites in 2023, up +327% from 554 the previous year. **Mixpanel** is deployed by 5,218 (0.5%), down -17% from 6,295 the previous year.
```

joethreepwood commented on contents/blog/mixpanel-alternatives.md:
```diff
@@ -0,0 +1,322 @@
+
+### Why do companies use PostHog?
+
+According to [reviews on G2](https://www.g2.com/products/posthog/reviews), companies use PostHog because:
```
```suggestion
According to [reviews on G2](https://www.g2.com/products/posthog/reviews), companies trust PostHog because:
```

Always make it emotive, imho

joethreepwood commented on contents/blog/mixpanel-alternatives.md:
```diff
@@ -0,0 +1,322 @@
+
+According to [reviews on G2](https://www.g2.com/products/posthog/reviews), companies use PostHog because:
+
+1. **It replaces multiple tools:** PostHog can replace Mixpanel (product analytics), LaunchDarkly (feature flags and A/B testing), and Hotjar (session replay and surveys). This simplifies workflows and ensures all their data is in one place.
```
```suggestion
1. **It replaces multiple tools:** PostHog can replace [Mixpanel](/blog/posthog-vs-mixpanel) (product analytics), LaunchDarkly (feature flags and A/B testing), and [Hotjar](/blog/posthog-vs-hotjar) (session replay and surveys). This simplifies workflows and ensures all their data is in one place.
```

joethreepwood commented on contents/blog/mixpanel-alternatives.md:
```diff
@@ -0,0 +1,322 @@
+
+> ### Bottom line 
+>
+> PostHog is the best Mixpanel alternative for startups and mid-size companies, especially those who would rather use one integrated platform instead of multiple, more expensive tools. It's also the best alternative for engineers.
```
We should clarify WHY it's best for engineers

joethreepwood commented on contents/blog/mixpanel-alternatives.md:
```diff
@@ -0,0 +1,322 @@
+
+### What is Google Analytics 4?
+
+Google Analytics 4 (GA4) is a marketing and product analytics tool that's tightly integrated with other Google products, such as Google Ads, BigQuery, Looker Studio, and Firebase. 
```
```suggestion
Google Analytics 4 (GA4) is a marketing and web analytics tool that's tightly integrated with other Google products, such as Google Ads, BigQuery, Looker Studio, and Firebase. 
```

joethreepwood commented on contents/blog/mixpanel-alternatives.md:
```diff
@@ -0,0 +1,322 @@
+
+> ### Bottom line
+>
+> GA4 is the best Mixpanel alternative for marketing teams who want to track ROI on campaigns.
```
I'd add here how it's suited to teams who only want to track data in specific ways too

andyvan-ph replied:
What do you mean by "specific ways" here?

joethreepwood replied:
As in, there are things you can't do in GA4 here. GA4 is the best Mixpanel alternative for marketing teams who want to track ROI data on Google ad campaigns, or simple web analytics metrics. It isn't suitable for those who want to explore deeper product metrics, or analyze data with SQL.

joethreepwood commented on contents/blog/mixpanel-alternatives.md:
```diff
@@ -0,0 +1,322 @@
+
+Heap describes itself as a Digital Insights Platform. Unpacking that a bit, it means Heap offers both product analytics and session replay, while also supporting marketing use cases with multitouch attribution.
+
+Heap was acquired in September 2023 by Contentsquare, marketing and e-commerce analytics firm, and announced plans to integrate the two products. 
```
```suggestion
Heap was acquired in September 2023 by Contentsquare, a marketing and e-commerce analytics firm, and announced plans to integrate the two products. 
```

andyvan-ph commented:
Approved @joethreepwood? I haven't even finished it yet 😅

ivanagas commented on contents/blog/best-mixpanel-alternatives.md:
```diff
@@ -0,0 +1,498 @@
+
+> ### Bottom line 
+>
+> PostHog is the best Mixpanel alternative for startups and mid-size companies. It replaces Mixpanel and numerous other tools, saving money and time. Power user features, like an SQL insight builder and console logs, make it a good choice for engineering-led teams, too.
```
```suggestion
> PostHog is the best Mixpanel alternative for startups and mid-size companies. It replaces Mixpanel and numerous other tools, saving money and time. Power user features, like an SQL insight builder and session replay logs, make it a good choice for engineering-led teams, too.
```
It is a bit unclear what "console logs" means here without some context.

ivanagas commented on contents/blog/best-mixpanel-alternatives.md:
```diff
@@ -0,0 +1,498 @@
+
+### Who uses GA4?
+
+Typical GA4 users are:
```
heavier ecommerce usage too?

ivanagas commented on contents/blog/best-mixpanel-alternatives.md:
```diff
@@ -0,0 +1,498 @@
+
+- Non-technical product managers
+- Marketing and design teams
+
```
companies focused on user journeys and funnels could be another one here

ivanagas commented on contents/blog/best-mixpanel-alternatives.md:
```diff
@@ -0,0 +1,498 @@
+
+## Google Analytics 4 (GA4)
+
+![GA4](../images/blog/mixpanel-alternatives/google-website.png)
```
This screenshot made me chuckle

andyvan-ph replied:
Haha, yes. I liked your website screenshots in the vs you wrote. Figured I'd let their marketing do the talking ;)

ivanagas commented on contents/blog/best-mixpanel-alternatives.md:
```diff
@@ -0,0 +1,498 @@
+ <td>Track website activity in real time</td>
+ </tr> 
+ <tr>
+ <td><strong>Data governance</strong></td>
```
GA has this too? Kind of: https://support.google.com/google-ads/answer/9305587?hl=en#zippy=%2Cgoogle-analytics

andyvan-ph replied:
I think that's more akin to user permissions. With data governance, an admin can approve or block new incoming events – e.g. in Mixpanel, if a new event is ingested, the admin gets notified and decide if it appears in the UI, add descriptions / tags etc.

ivanagas commented on contents/blog/best-mixpanel-alternatives.md:
```diff
@@ -0,0 +1,498 @@
+ <td className="text-center"><span className="text-red text-lg">✖</span></td> 
+ <td className="text-center"><span className="text-green text-lg">✔</span></td> 
+ <td>Admin level control over event approval, tags and descriptions</td>
+ </tr> 
```
Transparent pricing as another field here? Mixpanel yes, GA4 no?

ivanagas commented on contents/blog/best-mixpanel-alternatives.md:
```diff
@@ -0,0 +1,498 @@
+- Non-technical product managers
+- Data teams at large enterprises
+- Growth engineering and marketing teams
+
```
Companies with heavy regulatory requirements like banking and healthcare (at least that what it seems like from their page).

ivanagas commented on contents/blog/best-mixpanel-alternatives.md:
```diff
@@ -0,0 +1,498 @@
+2. **Large-scale experimentation:** Unlike Mixpanel, Amplitude offers a built-in experimentation add-on. This enables companies to run experiments on users using existing cohorts created in Amplitude.
+
+3. **Resolving data quality problems:** Companies that use Amplitude, particularly large ones, often migrate from outdated, or self-built, tools that generate poor quality data. Amplitude helps them fix that, while also making analytics more accessible.
+### How popular is Amplitude?
```
```suggestion

### How popular is Amplitude?
```

ivanagas reviewed (approved):
Looks good. Some ideas for the conclusion:
- Just a CTA is fine
- Check out PostHog paragraph CTA
- For a full conclusion, could do what does Mixpanel do well, why would someone want to switch?
